gate-core/src/main/groovy/com/netflix/spinnaker/gate/services/internal/KeelService.java

```diff
@@ -52,4 +52,10 @@
   @GET("/application/{application}")
   Map getApplicationDetails(
       @Path("application") String application, @Query("includeDetails") Boolean includeDetails);
+
+  @POST("/vetos/{name}")
+  Map passVetoMessage(@Path("name") String name, @Body Map<String, Object> message);
```

Comment:
this endpoint... does not return anything at all. Just a status code. Do we usually do something specific re: typing responses like that? I didn't see any examples and it seems like the approach historically has been "when in doubt, reach for `Map`.

Reply:
What happens now when you hit gate with a `veto` that doesn't exist?

Reply:
If the response is just a status code, use `Response<Void>` as the method return type.

Reply:
❤️ knew somebody would have the special sauce — thanks!

Reply:
@robzienert Which `Response` type were you referring to? I see we use `retrofit.client.Response` in some places, but that doesn't take a type parameter (and using it here serializes out a bunch of verbose response metadata like headers and whatnot). Poking around, the only similar type signatures I saw were the `okhttp3` one (also doesn't take a parameter) and the one from `jedis` we use in Kork/Clouddriver/etc.

I did finally find [an example of a Gate endpoint that only returns a status code](https://github.com/spinnaker/gate/blob/009124a5003709776374968728a79028decfa9a1/gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/V2CanaryConfigController.groovy#L67-L72), which appears to type the internal service client as a retrofit `Response`, and then use `void` as the return type in the controller (with a pit stop in a hysterix command that returns `void`). Is that closer to what you're looking for?

Reply:
aside: once https://github.com/spinnaker/keel/pull/446 is out hitting this with a bad veto name will send back a 404

Reply:
If you don't care about the response at all, which I guess you don't, then you can use `void`, otherwise you can use `retrofit.client.Response` to get the "raw" response object. It looks like `void` will suit you just fine.

Reply:
done! thanks rob.